ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -623,4 +635,20 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storePersistRemoteTokenIntervalInSeconds =
         verifiableProperties.getIntInRange(storePersistRemoteTokenIntervalInSecondsName, 0, 0, 60 * 60 * 24);
   }
+
+  /**
+   * Returns the delete-message retention time in minutes.
+   * This method helps to make the transition from StoreConfig#storeDeletedMessageRetentionHours to
+   * StoreConfig#storeDeletedMessageRetentionMinutes.
+   * If storeDeletedMessageRetentionMinutes is set to default value then it uses storeDeletedMessageRetentionHours.
+   *
+   * @return delete message retention time in minutes.
+   */
+   public int getStoreDeletedMessageRetentionInMinutes() {
```

Comment:
nit, we don't have to create a new method to return minute value, we can just use storeDeletedMessageRetentionMinutes. Just in the constructor, we can do 
```
long minuteValue =
    verifiableProperties.getIntInRange("store.deleted.message.retention.minutes", -1, -1, Integer.MAX_VALUE);
storeDeletedMessageRetentionMinutes = minuteValue == -1? storeDeletedMessageRetentionHours * 60: minuteValue;
```

Reply:
updated.